src/mod/applications/mod_dptools/mod_dptools.c

```diff
@@ -3235,6 +3237,7 @@ SWITCH_STANDARD_APP(loop_playback_function)
 		switch_channel_set_variable(channel, SWITCH_CURRENT_APPLICATION_RESPONSE_VARIABLE, "FILE NOT FOUND");
 		break;
 	default:
+		switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_ERROR, "Could not play the media correctly. status: %d, error: %s\n", status, error);
```

Comment:
Since you already have a session here, its better for you to log it so we can easily filter this. Use SWITCH_CHANNEL_SESSION_LOG(session) instead of SWITCH_CHANNEL_LOG

Reply:
Good point, fixed it. :)